content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -32,40 +32,52 @@ plane and the sidecars for the Istio data plane.
    instead with the flags `--set gateways.istio-ingressgateway.type=NodePort --set gateways.istio-egressgateway.type=NodePort`
    appended to the end of the helm operation.
 
-## Option 1: Install with Helm via `helm template`
+## Installation steps
 
-1. Render Istio's core components to a Kubernetes manifest called `istio.yaml`:
+1. Install Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the kube-apiserver:
+
+    {{< text bash >}}
+    $ kubectl apply -f install/kubernetes/helm/istio/templates/crds.yaml -n istio-system
+    {{< /text >}}
+
+1. Choose one of the following two
+**mutually exclusive** options described below.
+
+### Option 1: Install with Helm via `helm template`
+
+* Render Istio's core components to a Kubernetes manifest called `istio.yaml`:
 
     {{< text bash >}}
     $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system > $HOME/istio.yaml
     {{< /text >}}
 
-1. Install the components via the manifest:
+* Install the components via the manifest:
 
     {{< text bash >}}
     $ kubectl create namespace istio-system
     $ kubectl create -f $HOME/istio.yaml
     {{< /text >}}
 
-## Option 2: Install with Helm and Tiller via `helm install`
+### Option 2: Install with Helm and Tiller via `helm install`
 
 This option allows Helm and
 [Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components)
 to manage the lifecycle of Istio.
 
-1. If a service account has not already been installed for Tiller, install one:
+* If a service account has not already been installed for Tiller, install one:
```

Comment:
this is an ordered list, please keep the `1.`

Reply:
The problem was a could not do sub-numbered lists, and I already have Installation steps 1 and 2 above. If I transform this into a numbered list, it will be confusing with the steps above.

Reply:
I changed it nevertheless, so we can get this in sooner, people need to start testing with the new candidate image and new instructions.